preload.js

```diff
@@ -81,6 +92,27 @@ window.sessionFeatureFlags = {
   debugOnionRequests: false,
 };
 
+function parseProOriginatingPlatform(v) {
+  return v === 1 || v === 2 ? v : null;
+}
+
+function parseProAccessVariant(v) {
+  return v === 1 || v === 2 || v === 3 ? v : null;
+}
+
```

Comment:
why is this untyped? and can we have something with some meaning or this is what libsession is going to expose?

Reply:
needed a way to handle the env variable, but i think im going to remove the ability to set this from an env variable. Cant type it as its in a js file. What do you think? just remove it?

Reply:
removed